File: Platform.FoodMe.Admin/Platform.FoodMe.Admin/src/app/shared/Models/LocationData/CountryDto.ts
export enum Status {
  isDeleted = 0,
  isUpadate = 1,
  isActive = 2,
  isBlackListed = 3,
  isDefault = 4,
  isInactive = 5
}
export interface ReferentialDataBase {
  status: Status;
  creationTime: string | null;
  creatorUserId: number | null;
  deleterUserId: number | null;
  deletionTime: string | null;
  lastModifierUserId: number | null;
  lastModificationTime: string | null;
}
export interface CountryDTO extends ReferentialDataBase {
  id: number;
  nameLabelCode: string;
  code: string;
  countryKey: string;
}

File: Microservice/Platform.ReferencialData.WebAPI/Platform.ReferentialData.DtoModel/LocationData/CountryDTO.cs
using Platform.Shared.Enum;
using ProtoBuf;

namespace Platform.ReferentialData.DtoModel.LocationData
{
    [ProtoContract]
    public class CountryDTO
    {
        [ProtoMember(1)]
        public int Id { get; set; }
        [ProtoMember(2)]
        public Guid NameLabelCode { get; set; }
        [ProtoMember(3)]
        public string Code { get; set; }
        [ProtoMember(4)]
        public string CountryKey { get; set; }
        public Status Status { get; set; }

    }
}
